backend/src/models/User.ts

```diff
@@ -36,6 +64,11 @@ UserSchema.methods.isValidPassword = async function (
   return compare;
 };
 
-const User = mongoose.model<User, UserModel>("user", UserSchema);
```

Comment:
you need this

Reply:
we moved it up

Reply:
i would just move it down after the discriminators, up to you